From 48a73005df82654a848fcbe7344bcd8042a978e2 Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Tue, 12 Dec 2017 05:18:18 -0600
Subject: [PATCH] "0001-vold_fix_path_max_minors
 _______________________________

*Previous From 1aa6c09532bcaac95d9c5f7ecfa685c556daab25 Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Sat, 5 Nov 2016 11:19:26 -0500
*Previous Subject: [PATCH] vold_fix_path_max_minors
*
*Change-Id: I5af71df3370959047c9bde1d5f66e9c5a37ec0eb"
---
 Disk.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Disk.cpp b/Disk.cpp
index 829feae..35e6355 100644
--- a/Disk.cpp
+++ b/Disk.cpp
@@ -48,7 +48,7 @@ static const char* kSgdiskPath = "/system/bin/sgdisk";
 static const char* kSgdiskToken = " \t\n";
 
 static const char* kSysfsLoopMaxMinors = "/sys/module/loop/parameters/max_part";
-static const char* kSysfsMmcMaxMinors = "/sys/module/mmcblk/parameters/perdev_minors";
+static const char* kSysfsMmcMaxMinors = "/sys/module/mmc_block/parameters/perdev_minors";
 
 static const unsigned int kMajorBlockLoop = 7;
 static const unsigned int kMajorBlockScsiA = 8;
-- 
2.7.4

